packages/@lwc/engine-core/src/framework/vnodes.ts

```diff
@@ -165,6 +165,11 @@ export interface VElementData extends VNodeData {
     readonly slotData?: any;
     // Corresponds to the slot attribute of the element and indicates which `slot` element it should be assigned to
     readonly slotAssignment?: string;
+    readonly context?: {
+        lwc?: {
+            dom?: 'manual';
+        };
+    };
```

Comment:
Weird, I'm surprised this worked before. 🙃

Reply:
It was previously typed as `any`!